.vscode/configurationCache.log

```diff
@@ -0,0 +1 @@
+{"buildTargets":[],"launchTargets":[],"customConfigurationProvider":{"workspaceBrowse":{"browsePath":[],"compilerArgs":[]},"fileIndex":[]}}
```

Comment:
You should add .vscode to your gitignore so you don't have all these vscode files

Reply:
added to my git ignore